vizro-core/tests/unit/vizro/managers/test_model_manager.py

```diff
@@ -0,0 +1,208 @@
+from typing import Any
+
+import pytest
+
+import vizro.models as vm
+from vizro.managers import model_manager
+from vizro.managers._model_manager import FIGURE_MODELS
+
+
+@pytest.fixture
+def page_1(standard_px_chart):
+    return vm.Page(
+        id="page_1_id",
+        title="Page 1",
+        components=[vm.Button(id="page_1_button_id"), vm.Graph(id="page_1_graph_id", figure=standard_px_chart)],
+        controls=[],
+    )
+
+
+@pytest.fixture
+def managers_dashboard_two_pages(vizro_app, page_1, standard_kpi_card):
+    return vm.Dashboard(
+        pages=[
+            page_1,
+            vm.Page(
+                id="page_2_id",
+                title="Page 2",
+                components=[
+                    vm.Button(id="page_2_button_id"),
+                    vm.Figure(id="page_2_figure_id", figure=standard_kpi_card),
+                ],
+                controls=[vm.Filter(id="page_2_filter", column="year")],
+            ),
+        ]
+    )
+
+
+pytestmark = pytest.mark.usefixtures("managers_dashboard_two_pages")
+
+
+class TestGetModels:
+    """Test _get_models method."""
+
+    def test_model_type_none_page_none(self):
+        """model_type is None | page is None -> return all elements."""
+        result = [model.id for model in model_manager._get_models()]
+
+        expected_present = {
+            "page_1_id",
+            "page_1_button_id",
+            "page_1_graph_id",
+            "page_2_id",
+            "page_2_button_id",
+            "page_2_figure_id",
+        }
+
+        assert expected_present.issubset(result)
```

Comment:
Is there a reason why we check for the subset instead of equality? What else is inside result? 🤔 
```suggestion
        assert expected.issubset(result)
```

Reply:
>  What else is inside result?

Layout, Navigation and Dashboard objects are also returned. The real reason why I haven't checked the equality is that we can now extend the `managers_dashboard_two_pages` Dashboard without a need to change all tests.